api/kessel/inventory/v1beta2/check_self_response.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package kessel.inventory.v1beta2;
+
+import "kessel/inventory/v1beta2/allowed.proto";
+import "kessel/inventory/v1beta2/consistency.proto";
+option go_package = "github.com/project-kessel/inventory-api/api/kessel/inventory/v1beta2";
+option java_multiple_files = true;
+option java_package = "org.project_kessel.api.inventory.v1beta2";
+
+// CheckSelfResponse represents the result of a self-access permission check.
+message CheckSelfResponse {
+  Allowed allowed = 1;
+  optional Consistency consistency_token = 2;
```

Comment:
Similar idea here, should just return the `ConsistencyToken` type to get the token output to be inline with something like `StreamedListObjects`